proxy/bootstrap/src/main/java/org/apache/shardingsphere/proxy/Bootstrap.java

```diff
@@ -51,10 +52,8 @@ public static void main(final String[] args) throws IOException, SQLException {
         int port = bootstrapArgs.getPort().orElseGet(() -> new ConfigurationProperties(yamlConfig.getServerConfiguration().getProps()).getValue(ConfigurationPropertyKey.PROXY_DEFAULT_PORT));
         List<String> addresses = bootstrapArgs.getAddresses();
         new BootstrapInitializer().init(yamlConfig, port, bootstrapArgs.getForce());
-        boolean cdcEnabled = null != yamlConfig.getServerConfiguration().getCdc() && yamlConfig.getServerConfiguration().getCdc().isEnabled();
-        if (cdcEnabled) {
-            new CDCServer(addresses, yamlConfig.getServerConfiguration().getCdc().getPort()).start();
-        }
+        Optional.ofNullable((Integer) yamlConfig.getServerConfiguration().getProps().get(ConfigurationPropertyKey.CDC_SERVER_PORT.getKey()))
+                .ifPresent(cdcPort -> new CDCServer(addresses, cdcPort).start());
```

Comment:
Will start CDC server throw exception? If true, then it might need to catch and ignore it